ephios/settings.py

```diff
@@ -46,8 +46,8 @@
     X_FRAME_OPTIONS = "DENY"
     SECURE_CONTENT_TYPE_NOSNIFF = True
     SECURE_SSL_REDIRECT = True
-    SECURE_HSTS_SECONDS = 3600
-    SECURE_HSTS_INCLUDE_SUBDOMAINS = True
+    SECURE_HSTS_SECONDS = env.int("SECURE_HSTS_SECONDS", default=3600 * 24 * 366)
```

Comment:
I would prefer having an additional ENABLE_HSTS flag. That way, if HSTS is enabled, the default for the duration is sensible. On the other hand, defaulting the flag to false means that HSTS won't activate without the user understanding the consequences of it being set to one year/the server configured with improper SSL

Reply:
We could just default to hsts 3600 seconds and put a section in the deployment guide on ssl hardening

Reply:
We could just default to hsts 3600 seconds and put a section in the deployment guide on ssl hardening

Reply:
Why do you want 3600 seconds as default if there is significantly more recommended everywhere?

Reply:
We want a long hsts to be configured explicitly after people verified that their setup is working. Otherwise they might lock browsers out (because of how HSTS works).

Thats what recommended in the Django documentation as well.

Reply:
Don't get me wrong, we want admins to have a secure setup, and that includes a long HSTS, but copying numbers, shell commands and config files is not the full story here.